src/redux-middleware/scrollCursorIntoView.ts

```diff
@@ -27,7 +56,10 @@ const scrollCursorIntoViewMiddleware: ThunkMiddleware<State> = ({ getState }) =>
     // if the cursor has changed, scroll it into view
     const cursor = getState().cursor
     if (cursor !== cursorLast) {
-      scrollCursorIntoView()
+      // indicate that the cursor has changed and we want to scroll it into view
+      // this is needed for when thoughts need to be pulled from storage prior to scrolling
+      navigated = true
+      scrollCursorIntoView({ navigated: true })
```

Comment:
Setting `navigated: true` here is too general. In most cases, we want to scroll the cursor into view on the next animation frame, not after 100ms (really, as soon as the hierarchy re-renders).

Under what conditions do we need to delay 100ms? Did changing `requestAnimationFrame to `setTimeout(..., 0)` break anything?

Reply:
`requestAnimationFrame` and `setTimeout(..., 0)` worked the same, otherwise would have opted to stick with rAF in an if/else.